packaging/amazon-linux-ami-integrated/ecs-agent.spec

```diff
@@ -26,16 +26,15 @@ License:        Apache 2.0
 Summary:        Amazon Elastic Container Service initialization application
 # TODO: add back aarch64
 ExclusiveArch:  x86_64
- 
-Source0:        sources-with-agent.tgz
+
+Source0:        sources.tgz
 Source1:        ecs.conf
 Source2:        ecs.service
 Source3:        amazon-ecs-volume-plugin.service
 Source4:        amazon-ecs-volume-plugin.socket
 Source5:        amazon-ecs-volume-plugin.conf
 
 BuildRequires:  golang >= 1.7
-BuildRequires:  glibc-static
```

Comment:
Was this dependency required only for building pause container?

Reply:
Yeah, it was.